docs/extend/plugins.md

```diff
@@ -65,6 +65,12 @@ Plugin
 [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume)                   | A volume plugin that connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
+### Authorization plugins
+
+| Plugin                                                       | Description                                                                                                                                                                                           |
+|--------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Twistlock AuthZ Broker](https://github.com/twistlock/authz) | A basic extendable Docker authorization plugin that runs on directly on the host or inside a container. It basically allows you to define a few user policies that it evaluates during authorization. |
```

Comment:
Perhaps something like;

A basic extendable authorization plugin that runs directly on the host or inside a container. This plugin allows you to define user policies that it evaluates during authorization. Basic authorization is provided if Docker daemon is started with the `--tlsverify` flag (username is extracted from the certificate common name).